remo/events/templates/view_event.html

```diff
@@ -299,24 +299,37 @@ <h1 class="event-single-title">{{ event.name }}</h1>
     </div>
   </div>
   <!-- END Mashup code -->
-
-  {% macro display_attendee(attendee) -%}
-    <div class="event-single-attendee">
-      <div class="row">
-        <div class="large-3 small-3 columns grid-profile-image">
-          <img src="{{ attendee|get_avatar_url }}"
-               class="profiles-people-avatar"
-               alt="Avatar">
-        </div>
-        <div class="large-9 small-9 columns grid-profile-text">
-          <h6>{{ attendee.get_full_name() }}</h6>
-          <p class="event-single-attendee-text">
-            {{ get_attendee_role_event(attendee, event) }}
-          </p>
+  {% if attendee.count >= 3 %}
+    <div class="row">
+    {% macro display_attendee(attendee) -%}
```

Comment:
There is no need to define a second macro here. You can use the `if/else` statement inside the existing macro.